src/cpu.h

```diff
@@ -57,6 +57,9 @@ void cpu_step(void);
 __attribute__((pure)) uint16_t cpu_get_PC(void);
 void cpu_print_registers(void);
 
+__attribute__((pure)) uint8_t get_flag_bit(Flag f);
+void set_flag(uint8_t bit, Flag f);
```

Comment:
I think that both of these should be `always_inline` since that allows us to switch a `CALL` instruction with `AND` or something similar when optimization is < O2